Model/Currency.hs

```diff
@@ -1,3 +1,9 @@
+{-# LANGUAGE CPP #-}
+
+#if MIN_VERSION_base(4,8,0)
+{-# LANGUAGE FlexibleContexts #-}
```

Comment:
We already have lots of code with FlexibleContexts, a short grep away. :)

On Fri, May 1, 2015 at 8:53 AM, Peter Harpending <notifications@github.com>
wrote:

> In Model/Currency.hs
> <https://github.com/snowdriftcoop/snowdrift/pull/305#discussion_r29510004>
> :
>
> > @@ -1,3 +1,9 @@
> > +{-# LANGUAGE CPP #-}
> > +
> > +#if MIN_VERSION_base(4,8,0)
> > +{-# LANGUAGE FlexibleContexts #-}
>
> @nkaretnikov <https://github.com/nkaretnikov> Do you have a source for
> that
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/snowdriftcoop/snowdrift/pull/305/files#r29510004>.
>